utils/test-deployment/Dockerfile

```diff
@@ -1,6 +1,21 @@
-FROM python:3.8-slim
+FROM debian:11-slim AS build-stage
+WORKDIR /venv
+RUN apt-get update && \
+    DEBIAN_FRONTEND=noninteractive apt-get install \
+    --no-install-suggests --no-install-recommends --yes \
+    python3-venv=3.9.2-3 && \
+    python3 -m venv /venv
 
-COPY utils/test-deployment .
-RUN pip install -r requirements.txt
+COPY utils/test-deployment/requirements.txt /requirements.txt
+RUN /venv/bin/pip install --disable-pip-version-check --requirement /requirements.txt
 
-CMD [ "./run_tests.py" ]
\ No newline at end of file
+FROM gcr.io/distroless/python3-debian11 as production-stage
```

Comment:
This is considered experimental. We could take the risk, it just has to be a known risk.

![image](https://user-images.githubusercontent.com/57442769/183380886-e21bdf83-c306-423e-aa2a-c672d515e570.png)

Reply:
Yes, I'd noticed that.  I wanted to ensure that the things that run python (e.g. fileimporter) were all working before taking this PR out of draft status.

Slightly more details on why it is marked as experimental: 
> special requirements (like a shell) and another included binary ([ld.so.amd64.cache](https://github.com/GoogleContainerTools/distroless/blob/main/experimental/python3/ld.so.amd64.cache)) which have it sitting in experimental -- we also don't have the current capability to enforce build preconditions on arm machines

https://github.com/GoogleContainerTools/distroless/issues/973#issuecomment-1065169760

Assuming this is everything that makes it experimental; I'd argue that we're currently running python in an image with a shell (and presumably the mentioned binary) so distroless is no worse than our current state.  We're also not running on arm architecture in production (and I assume we don't intend to support that).